generators/create-react-app/templates/base/.eslintrc.json

```diff
@@ -13,6 +13,10 @@
             "env": {
                 "jest": true
             }
+        },
+        {
+          "files": ["next.config.js"],
+          "parser": "espree"
```

Comment:
The indentation is wrong here.

Reply:
And it's in the wrong file, I don't think a configuration for the Next config file is usefull in the CRA generator :rofl:

Reply:
🤦 rectified.